lib/parallel.rb

```diff
@@ -31,10 +28,14 @@ class ExceptionWrapper
     attr_reader :exception
     def initialize(exception)
       @exception =
-        begin
-          Marshal.dump(exception) && exception
-        rescue
-          UndumpableException.new(exception)
+        if [Parallel::Break, Parallel::Kill].include? exception.class
+          exception.class.new
```

Comment:
Yes, I was thinking that we don't use it, and these exceptions are never visible to the developer